README.md

```diff
@@ -34,12 +139,45 @@ and set this plugin in settings.
 **Step 3:** Add the plugin to plugins list:
```

Comment:
Too many steps to describe a common operation. Find a way to make it shorter. For example, just mention the commands & give clear instructions.

You don't need to say `check your project for existing PostCSS config`. It is kind of obvious thing to say. Instead, say `add bla bla bla to your postcss config, like below`.

Reply:
Yeah, this is what I wanted to discuss. The first section came with the boilerplate. I wan't to skip it completely and just keep the `programmatic` section.

Reference usage sections in some plugins:
https://github.com/2createStudio/postcss-sprites#usage
https://github.com/postcss/postcss-custom-properties#usage
https://github.com/pascalduez/postcss-apply#usage